page.css

```diff
@@ -0,0 +1,506 @@
+@font-face {
+  font-family: cocogoose;
+  src: url(fonts/Cocogoose-font.otf);
+}
+
+html,
+body {
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  font-size: 16px;
+  font-family: "lato", sans-serif;
+}
+
+p {
+  line-height: 1.5rem;
+}
+
+.menu-content {
+  display: none;
+}
+
+.top-bar-section {
+  background-color: black;
+  display: none;
+}
+
+.navigation-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  width: 95%;
+}
+
+.nav-toggle {
+  position: absolute;
+  top: 1rem;
+  cursor: pointer;
+  left: 1rem;
+}
+
+.close-menu {
+  display: none;
+  cursor: pointer;
+}
+
+.close-menu p {
+  font-size: 3.2rem;
+  position: absolute;
+  margin: 0;
+  top: 0;
+  left: 1.3rem;
+}
+
+.logo-desktop a {
+  display: none;
+  position: absolute;
+  top: 1.3rem;
+  right: 1.5rem;
+  text-decoration: none;
+  color: black;
+}
+
+.pages-menu {
+  width: 100%;
+  display: none;
+  height: 80vh;
+  background-color: rgb(234, 234, 234, 0.7);
+  border-right: 4px outset #ec5242;
+  border-bottom: 2px outset #ec5242;
+}
```

Comment:
- [ ] There is a problem with the height, and it's an it's the height and width of the border line, you can fix this with a height: 100vh and a width: 100%

Your design problem:

![imagen](https://user-images.githubusercontent.com/60631456/128283764-6eeb4349-40c7-4d89-8351-edd359fbd00f.png)